go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/celestiaorg/celestia-app/v4
 go 1.23.6
 
 require (
-	cosmossdk.io/api v0.7.6
+	cosmossdk.io/api v0.9.0
```

Comment:
https://github.com/cosmos/cosmos-sdk/blob/api/v0.9.0/api/CHANGELOG.md

Is this the correct changelog to be looking at cc @julienrbrt ? If so, LGTM b/c no breaking changes called out in v0.9.0. Unclear if there were any in v0.8.0 because they're not in that changelog.

Reply:
might be fine, not 100%, but this version matrix suggests v0.7.x https://github.com/cosmos/cosmos-sdk/blob/release/v0.52.x/README.md#core-dependencies

Reply:
Thanks for the link.

Tangent: It's poor maintainer dev UX to reference this version matrix to figure out which versions of cosmos sdk packages are compatible with other versions of cosmos sdk packages. I would expect v0.9.x to remain compatible because it's just a minor version bump. If it's not compatible with this Cosmos SDK version then why not cut a new major version of cosmossdk.io/api?